zerver/lib/send_email.py

```diff
@@ -207,6 +207,7 @@ def send_email(
     language: Optional[str] = None,
     context: Dict[str, Any] = {},
     realm: Optional[Realm] = None,
+    dry_run: Optional[bool] = False,
```

Comment:
```suggestion
    dry_run: bool = False,
```
See https://zulip.readthedocs.io/en/latest/testing/mypy.html#use-optional-and-none-correctly